src/main/java/io/cdap/plugin/servicenow/connector/ServiceNowConnectorConfig.java

```diff
@@ -172,17 +231,23 @@ public Double convertToDoubleValue(Object fieldValue) {
     if (fieldValue instanceof String && Strings.isNullOrEmpty(String.valueOf(fieldValue))) {
       return null;
     }
-
-    return Double.parseDouble(String.valueOf(fieldValue));
+    try {
+      return NumberFormat.getNumberInstance(java.util.Locale.US).parse(String.valueOf(fieldValue)).doubleValue();
+    } catch (ParseException e) {
+      return null;
```

Comment:
this seems like a bad idea. If invalid values are read as null, there is no way to differentiate between an actual null and an bad value.

Reply:
still have this comment, do not do this anywhere. It is wrong behavior and will lead to data loss.